Makefile

```diff
@@ -11,8 +11,13 @@ all:
 	cp .build/debug/protoc-gen-swiftgrpc .
 	
 project:
-	swift package $(CFLAGS) generate-xcodeproj
-	@-ruby fix-project-settings.rb || echo "Consider running 'sudo gem install xcodeproj' to automatically set correct indentation settings for the generated project."
+	swift package $(CFLAGS) generate-xcodeproj --output SwiftGRPC.xcodeproj
+	@-ruby fix-project-settings.rb SwiftGRPC.xcodeproj || echo "Consider running 'sudo gem install xcodeproj' to automatically set correct indentation settings for the generated project."
+
+project-carthage:
+	swift package generate-xcodeproj --output SwiftGRPC-Carthage.xcodeproj
+	ruby fix-indentation-settings.rb SwiftGRPC-Carthage.xcodeproj || echo "You may need to install xcodeproj ('sudo gem install xcodeproj')!"
```

Comment:
Use @- for those as well?

Reply:
+1, except for `remove-unwanted-targets-for-carthage.rb` that should not be ignored in case of failure, I believe.